libs/ui-toolkit/src/components/dropdown-menu/dropdown-menu.stories.tsx

```diff
@@ -56,13 +50,13 @@ export const CheckboxItems = () => {
 
 export const RadioItems = () => {
   const files = ['README.md', 'index.js', 'page.css'];
-  const [file, setFile] = useState(files[1]);
+  const [selected, setSelected] = useState(files[1]);
 
   return (
     <div style={{ textAlign: 'center', padding: 50 }}>
       <DropdownMenu>
         <DropdownMenuTrigger>
-          <Button appendIconName="chevron-down">Open</Button>
+          <span>Open</span>
         </DropdownMenuTrigger>
         <DropdownMenuContent>
           <DropdownMenuItem inset onSelect={() => console.log('minimize')}>
```

Comment:
chore (non-blocking): instead of console.log maybe we can use an action('minimize') here. but no big deal, not part of your changes